command/pr.go

```diff
@@ -303,6 +304,10 @@ func prList(cmd *cobra.Command, args []string) error {
 			prNum = "#" + prNum
 		}
 		table.AddField(prNum, nil, colorFuncForPR(pr))
+		if !table.IsTTY() {
+			table.AddField(pr.State, nil, nil)
+			table.AddField(prReadyState(&pr), nil, nil)
```

Comment:
The two extra columns when output is redirected makes this output to be radically different from the human-readable one. However, I agree that we used to use color to communicate open/closed/merged/draft states and that we now have to communicate that using words as well. Should we add the same columns to the default terminal output as well, for parity?

Reply:
I collapsed "OPEN" and draft status into a combined "DRAFT" and moved the new column to the end of the output :+1: